pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -790,6 +736,55 @@ func (r *ReconcileHyperConverged) detectTaintedConfiguration(req *common.HcoRequ
 	}
 }
 
+func (r *ReconcileHyperConverged) updateHCO(req *common.HcoRequest) error {
```

Comment:
we're not updating HCO - HC operator, but HC CR. Pleasse consider renaming.

Reply:
Good point. Will rename.